### DISCUSSION TITLE
No IP shows

### DISCUSSION THREAD

**Discussion by NIDcornifleur** [mcguirepr89/BirdNET-Pi#476] [General]
Hello...

At home, everything was working perfectly... now, at the cottage, I spent the day trying to make the birdnetpi.local/ page appear without success. However, I can enter the system (I have a TV and an HDMI cable), changed the GPS coordinates but nothing to do, the IP (192.168.41.1) does not work. I can connect on BirdNET-PI but in the properties, no IP is displayed and impossible to connect in SSH mode as well as on birdnetpi.local/ nor birdnetpi.192.168.42.1 (after having added the IP in birdnet.conf)
There is a router here... I connected the CM4 with an ethernet cable, got an IP on the local network but unable to open the web page or in SSH.
Any ideas please?

![Untitled](https://user-images.githubusercontent.com/20214731/182042420-e018d1a6-adb1-4115-8c22-0183d35aade2.png)

**Comment by ehpersonal38** [comment]
Is your phone/computer that you're trying to ssh from connected to the same router as the Pi?

I ask because that screenshot looks strange to me, it shows the BirdNET-Pi acting as it's own wireless network? Can you clarify what's happening there?

**Comment by NIDcornifleur** [comment]
Having no success with the router, I try to connect directly to the birdnet access point 
![2](https://user-images.githubusercontent.com/20214731/182044632-a5f66930-27f8-4780-b9e1-9872a2964aeb.png)

**Reply by ehpersonal38** [reply]
Looks like you're using the image with Hotspot then? Will defer to @mcguirepr89 on this one

**Reply by matthew73210** [reply]
When you were connected via ethernet, what was the IP of the birdnet-pi? I often use [fing](https://www.fing.com/products/fing-app) on my phone to scan the whole network.  Just a heads up if you connect to the rpi hotspot the old IP won't work (or highly unlikely as its a different network)

**Reply by NIDcornifleur** [reply]
I use Fing regularly because I have (at home) a fairly complicated network with half a dozen Rpi's running and supporting all sorts of experiments.  
On the router, I had 192.168.2.18 but I could not connect in any way, neither by ethernet nor by wifi. And, via the birdnet access point, no IP was displayed and, of course, 192.168.42.1 was not connecting. Normally, I would not have insisted and simply reinstalled the whole thing but I am not equipped to do so, being more than a thousand km from home. 
Now I have a doubt... Does birdnet-pi, if it is no longer connected on a local network, become available as an access point? Or maybe I'm confusing it with another application... 
Thanks for your efforts, it is appreciated.

**Reply by nilspupils** [reply]
Hi, any chance you installed BirdNET-Pi via the "pre-installed image"? It has a built-in hotspot feature, please see the documentation here:
https://github.com/mcguirepr89/BirdNET-Pi/discussions/353

**Reply by NIDcornifleur** [reply]
Yes, it is installed with the image... I can see BirdNET-Pi in the list of available hot spots, I can select it and connect to it but I can't open the birdnetpi.local/ page. I am going to reinstall everything. The challenge is to find an adapter for microSd.

**Reply by matthew73210** [reply]
once you are connected to said hotspot, what info do you get when you do a fing scan?

**Comment by NIDcornifleur** [comment]
Nothing relevant to birdnet when I scan with Fing.  I have a wifi dongle on my computer in addition to the wifi of the device.  I can connect to birdnet but no ip available. 
![2](https://user-images.githubusercontent.com/20214731/182161524-49d7ac5e-d086-4245-9860-efedec44fe46.png)
![Untitled](https://user-images.githubusercontent.com/20214731/182161526-ce89f0f0-3b57-43d9-90e8-b93970b5f27e.png)

**Reply by nilspupils** [reply]
Have you tried
ssh birdie@birdnetpi
when connected to the network?

**Comment by NIDcornifleur** [comment]
Yes, nilspupils... without success.   
Finally found a SD microSD adapter and installed the BirdNET image. Everything works now. I can only thank you for your help. I'm very happy now !
